Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -24,8 +24,9 @@
 #include "BaseTarget.xcconfig"
 #include? "<DEVELOPER_DIR>/AppleInternal/XcodeConfig/AppleFeatures.xcconfig"
 
-GCC_PREPROCESSOR_DEFINITIONS = $(ENABLE_RELOCATABLE_WEBPUSHD) $(inherited) $(GCC_PREPROCESSOR_DEFINITIONS_$(ENABLE_WEBGPU_SWIFT));
+GCC_PREPROCESSOR_DEFINITIONS = $(ENABLE_RELOCATABLE_WEBPUSHD) $(inherited) $(GCC_PREPROCESSOR_DEFINITIONS_$(ENABLE_WEBGPU_SWIFT)) $(GCC_PREPROCESSOR_DEFINITIONS_$(ENABLE_SWIFT_DEMO_URI_SCHEME));
 GCC_PREPROCESSOR_DEFINITIONS_ENABLE_WEBGPU_SWIFT = ENABLE_WEBGPU_SWIFT;
+GCC_PREPROCESSOR_DEFINITIONS_ENABLE_SWIFT_DEMO_URI_SCHEME = ENABLE_SWIFT_DEMO_URI_SCHEME;
```

Comment:
nit: I think the ENABLE_WEBGPU_SWIFT one is needlessly wordy. Since build-webkit feature flags are defined as their own feature name, we can do this: 
```suggestion
GCC_PREPROCESSOR_DEFINITIONS = $(ENABLE_RELOCATABLE_WEBPUSHD) $(inherited) $(ENABLE_WEBGPU_SWIFT) $(ENABLE_SWIFT_DEMO_URI_SCHEME);
```

Reply:
Worked well, thanks!